actions/run-capslock/action.yaml

```diff
@@ -0,0 +1,59 @@
+name: Capslock compare
+description: Capslock compare
+
+inputs:
+  go-version:
+    default: "1.24.6"
+    required: true
+    description: "Go version used"
+  capslock-version:
+    default: "v0.2.8"
+    required: true
+    description: "Capslock version used"
+  scope:
+    default: ""
+    required: true
+    description: "The scope of analysis"
+  main-branch:
+    default: "main"
+    required: true
+    description: "Name of the main branch"
+  output-place:
+    default: "pr-comment"
+    required: true
+    description: "Output place"
+runs:
+  using: composite
+  steps:
+    - uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 # v5.5.0
+      with:
+        go-version: "${{ inputs.go-version }}"
+    - name: Cache capslock
+      id: capslock-cache
+      uses: actions/cache@0400d5f644dc74513175e3cd8d07132dd4860809 # v4.2.4
+      with:
+        path: ~/go/bin
```

Comment:
nit: would it be enough to cache only that one binary instead of the whole folder?